.ci/azure/old-docs.yml

```diff
@@ -0,0 +1,65 @@
+jobs:
+  - job:
+    displayName: Deploy Docs
+    pool:
+      vmImage: ubuntu-latest
+    variables:
+      python.version: "3.8"
+    timeoutInMinutes: 240
+    steps:
+      # Checkout simpeg repo, including tags.
+      # We need to sync tags and disable shallow depth in order to get the
+      # SimPEG version while building the docs.
+      - checkout: self
+        fetchDepth: 0
+        fetchTags: true
+        displayName: Checkout repository (including tags)
+
+      - script: |
+          git config --global user.name ${GH_NAME}
+          git config --global user.email ${GH_EMAIL}
+          git config --list | grep user.
+        displayName: "Configure git"
+        env:
+          GH_NAME: $(gh.name)
+          GH_EMAIL: $(gh.email)
+
+      - bash: echo "##vso[task.prependpath]$CONDA/bin"
+        displayName: Add conda to PATH
+
+      - bash: .ci/azure/setup_env.sh
+        displayName: Setup SimPEG environment
+
+      - script: |
```

Comment:
Should remove this step since it uploads to pypi in the other file.

Reply:
Right! I forgot to update the new version of the `old-docs.yml` as well 🤦🏼‍♂️